LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 SurveyMonkey
```

Comment:
I believe the name of the company should be formatted as such:

https://github.com/aconrad/pycobertura/blob/master/LICENSE.txt#L3